stable/agent/values.yaml

```diff
@@ -62,6 +62,16 @@ registryCreds:
   # to be used with imagePullSecrets
   dockerconfigjson: ""
 
+# Add scriptes to /docker-entrypoint.d/ via a ConfigMap
+entrypointd: {}
+# 01-install-kubectl: |
+#   #!/bin/bash
+#   set -euo pipefail
+#   apt-get update && apt-get install -y kubectl
+
+serviceAccount:
+  annotations: {}
+
```

Comment:
unrelated to this commit but worth *fixing a bug* and setting the `dind` map to `{}` below  ...vvv...?

 (can definitely make the adjustment in another PR but was thinking it could make sense here since this is performing multiple chart improvements)

Reply:
yes, it should be fixed in this PR, it got lost porting 4 PRs to 1.

Reply:
nice, thx

Reply:
@tjwallace can add that ^^ to this PR? thanks

Reply:
will do

Reply:
If we set `dind: {}` won't expressions like `if .Values.dind.enabled` fail?

Reply:
it will

Reply:
Then I'm confused on how this will fix a bug 🤔 Or I'm confused on what change to make 😂

Reply:
Good call @tjwallace and yea, no worries on the `dind.*` changes. I think I confused how values were being merged with user overrides.